docs/snippets/about-functions.md

```diff
@@ -0,0 +1,12 @@
+<!-- Snippet used in the following topics:
+- /docs/functions/README.md
+-->
+Knative Functions provides a simple programming model for using functions on Knative, without requiring in-depth knowledge of Knative, Kubernetes, containers, or dockerfiles.
+
+Knative Functions enables you to easily create, build, and deploy stateless, event-driven functions as Knative Services by using the `func` CLI.
+
+Functions can be written in any language supported by the available [language packs](https://github.com/knative/func/blob/main/docs/language-pack-providers/language-pack-contract.md){target=_blank}.
```

Comment:
I think we should mention the built in templates here, before language packs are introduced. The built-in templates are Go, Node.js, Python, Quarkus, Rust and TypeScript. Those are what I reference in the first section here https://deploy-preview-5243--knative.netlify.app/docs/functions/runtimes/